packages/carbon-react/package.json

```diff
@@ -35,7 +35,8 @@
   },
   "peerDependencies": {
     "react": "^16.8.6 || ^17.0.1",
-    "react-dom": "^16.8.6 || ^17.0.1"
+    "react-dom": "^16.8.6 || ^17.0.1",
+    "sass": ">= 1.32"
```

Comment:
Would we want a caret range instead so a new major wouldn't be an acceptable peer dep?

Reply:
Hell yeah, that makes a lot of sense.